terraform-post-packer/variables.tf

```
@@ -4,6 +4,12 @@
 # These parameters have reasonable defaults.
 # ------------------------------------------------------------------------------
 
+variable "ami_share_account_name_regex" {
+  default     = "^env[[:digit:]]+$"
```

Until we have renamed our production accounts from their "legacy" naming scheme, we must use a looser regex (e.g. see https://github.com/cisagov/kali-packer/commit/4a040018df95bbcb992fc311c5a71c9867676886):
```suggestion
  default     = "^env[[:digit:]]+"
```